include/Food.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef FOOD_H
+#define FOOD_H
+
+#include "../Util/Random.h"
```

Comment:
try to google target_include_directories rule for CmakeFile to get rid of this relative path: ../